### ISSUE TITLE
Reflection warnings

### ISSUE THREAD

**Issue by FieryCod** [metosin/jsonista#50]
I'm receiving some reflection warnings, when using jsonista. I think that changing this is reasonable, because some libraries which use jsonista with GraalVM now have to provide`reflection-config.json`. 

```
Reflection warning, jsonista/core.clj:139:19 - call to method registerModule can't be resolved (target class is unknown).
Reflection warning, jsonista/core.clj:140:19 - call to method registerModule can't be resolved (target class is unknown).
Reflection warning, jsonista/core.clj:142:39 - call to method enable can't be resolved (target class is unknown).
Reflection warning, jsonista/core.clj:143:44 - call to method enable can't be resolved (target class is unknown).
Reflection warning, jsonista/core.clj:144:71 - reference to field getFactory can't be resolved.
Reflection warning, jsonista/core.clj:144:71 - call to method enable can't be resolved (target class is unknown).
Reflection warning, jsonista/core.clj:146:8 - call to method registerModule can't be resolved (target class is unknown).
Reflection warning, jsonista/core.clj:146:8 - call to method registerModule can't be resolved (target class is unknown).
Reflection warning, jsonista/core.clj:147:6 - call to method disable can't be resolved (target class is unknown).
```

Would you like to have a PR for fixing this?

**Comment by viesti** [comment]
Thank you for noticing! I think a PR is very well welcome :)

**Comment by FieryCod** [comment]
It's already fixed there: https://github.com/metosin/jsonista/commit/90f447aac858a18194dcd95f01ca40062f41866f
Could you please roll out a new release :)?

**Comment by viesti** [comment]
Released in 0.3.2 @FieryCod !
